sdks/python/apache_beam/examples/snippets/transforms/element_wise/pardo.py

```diff
@@ -29,7 +29,8 @@ def __init__(self, delimiter=','):
       self.delimiter = delimiter
 
     def process(self, text):
-      return text.split(self.delimiter)
+      for word in text.split(self.delimiter):
```

Comment:
Why this change?

Reply:
I just thought it would be clearer to read and more consistent with other samples, although it's doing the same thing.

Reply:
Should we revert this?